independent-projects/tools/registry-client/src/test/resources/catalog-config/platforms.json

```diff
@@ -0,0 +1,13 @@
+{
+  "platforms" : [ {
+    "platform-key" : "io.quarkus",
+    "streams" : [ {
+      "id" : "999-SNAPSHOT",
+      "releases" : [ {
+        "version" : "999-SNAPSHOT",
+        "member-boms" : [ "io.quarkus:quarkus-bom::pom:999-SNAPSHOT" ],
+        "quarkus-core-version" : "999-SNAPSHOT"
```

Comment:
it is a file to parse. If we want, we could/should add some other variants here.

Reply:
We should be careful not to make our tests to actually depend on 999-SNAPSHOT. Which is why I previously asked Andy to use something else, this is how the "FAKE" suffix appeared.

Reply:
Used here: https://github.com/quarkusio/quarkus/blob/50b122aa946253c37c791f36fa65a4fa8615d75d/independent-projects/tools/registry-client/src/test/java/io/quarkus/registry/catalog/ExtensionCatalogSerializationTest.java#L33

Reply:
it is a file that is read and compared against a Platforms object. It could contain any old thing, but I happened to choose a file that we create in other tests down the line to make sure that I would produce the expected output.